Dockerfile

```diff
@@ -7,4 +7,4 @@ RUN npm install express path mustache-express promise prom-client dotenv jsdom r
 
 EXPOSE 8080
 
-CMD ["npm", "start"]
+CMD ["node", "./server.js"]
```

Comment:
det *kan* bli issues i gcp siden appen starter opp med å hente dekoratør fra internett (og kræsjer om den ikke får det til?). jeg tror at scuttle bør involveres i en egen dockerfile for gcp, slik at vi ikke blander det med onprem-greier

Reply:
Funker fint slik det er nå. Om den fetchen feiler tror jeg podden bare restartes. Allerede heftig mye endringer her, så kan mekkes senere. Vil isåfall da helst fortsatt bare ha et image, men disable scuttle i sbs clusterne. Så mye styr med to.

Reply:
fair, vi gjør det sånn enn så lenge!